builtin/providers/aws/resource_aws_route53_record.go

```diff
@@ -748,3 +749,16 @@ func nilString(s string) *string {
 	}
 	return aws.String(s)
 }
+
+func normalizeAwsAliasName(alias interface{}) string {
+	input := alias.(string)
+	if strings.HasPrefix(input, "dualstack.") {
+		return strings.Replace(input, "dualstack.", "", -1)
+	}
+
+	if strings.HasSuffix(input, ".") {
+		return input[0 : len(input)-1]
```

Comment:
could either of these operations be replaced with `strings.TrimLeft` ? https://golang.org/pkg/strings/#TrimLeft ?